### ISSUE TITLE
Add MiniImagenet to the Benchmarks

### ISSUE THREAD

**Issue by vlomonaco** [ContinualAI/avalanche#154]
We should add MiniImagenet to the benchmarks. It's been used here: https://arxiv.org/abs/1902.10486

**Comment by vlomonaco** [comment]
@Mattdl can you add the "classic" benchmark based on your latest paper maybe? :)

**Comment by Mattdl** [comment]
We actually used the preprocessed MiniImagenet data in an AvalancheDataset, but as @lrzpellegrini mentioned, there is no official download URL for this preprocessed data.

**Comment by vlomonaco** [comment]
Yes, but I was asking about the "[classic](https://avalanche-api.continualai.org/autoapi/avalanche/benchmarks/classic/?highlight=classic%20benchmarks#module-avalanche.benchmarks.classic)" benchmark, not the dataset itself. Since you have more experience with it maybe you can propose an high-level method to easily generate that benchmark.

**Comment by Mattdl** [comment]
I'll first take a look to see if I can be of help there (: I'll keep you posted here.

**Comment by Mattdl** [comment]
Due to prioritizing some other open issues and NeurIPS coming soon, I probably won't have time until the NeurIPS deadline.. Feel free to assign this one to another enthusiast (:
